src/actions/profile-view.js

```diff
@@ -46,7 +46,7 @@ import type { State } from '../types/state';
 import type { Action, ThunkAction } from '../types/store';
 import type {
   ThreadIndex,
-  IndexIntoMarkersTable,
+  IndexIntoRawMarkersTable,
```

Comment:
Should it be `IndexIntoRawMarkerTable` instead of `IndexIntoRawMarkersTable`? (no `s` at the end of `Marker`) because `MarkersTable` was renamed to `RawMarkerTable`?